.systemtest.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Nope, within `system-test.json` you can specify where to find the executables, but having the entire `Systemtests` file executable sounds like a better idea 👍